tests/EdgeToEdge/Routes/ApplyForMembershipRouteTest.php

```diff
@@ -446,4 +449,14 @@ public function testCookieFlagsSecureAndHttpOnlyAreSet(): void {
 		$this->assertTrue( $cookie->isSecure() );
 	}
 
+	public function testGivenDonationReceiptOptOutRequest_applicationHoldsThisValue(): void {
+		$this->createEnvironment( [], function ( Client $client, FunFunFactory $factory ): void {
+			$parameters = $this->newValidHttpParameters();
+			$parameters['donationReceipt'] = '0';
+
+			$client->request( Request::METHOD_POST, self::APPLY_FOR_MEMBERSHIP_PATH, $parameters );
+
+			$this->assertFalse( $factory->getMembershipApplicationRepository()->getApplicationById( 1 )->getDonationReceipt() );
```

Comment:
`getDonationReceipt` sounds like it returns a donation receipt

Reply:
Went with consistent naming in [Donation](https://github.com/wmde/FundraisingStore/commit/c8308cb8df6af7045ff0ee69b73c013da0e9b79d#diff-cd62f2322362fb9135726ef3359173e8R268) and Membership. Thinking about this led to [your other question](https://github.com/wmde/FundraisingFrontend/pull/976#issuecomment-339801033) - the answer to which would have blown things out of proportion. I think this is a good task for when the store parts get moved into the contexts.

Reply:
Oh wait, forgot in my last comment the name we're talking about is defined in the store component. Still think I'd have been better to go with another name, but indeed not an issue in this PR. I just wanted to point out I'm not 100% happy with this.